xline/src/client/mod.rs

```diff
@@ -68,11 +68,12 @@ impl Client {
         use_curp_client: bool,
         timeout: ClientTimeout,
     ) -> Result<Self, ClientError> {
+        let name = String::from("client");
         let etcd_client =
             EtcdClient::connect(all_members.values().cloned().collect_vec(), None).await?;
-        let curp_client = CurpClient::new(all_members, timeout).await;
+        let curp_client = CurpClient::new(name.clone(), all_members, timeout).await;
```

Comment:
The server id should be optional, otherwise, it will be confusing to see that `fetch_local_leader_info` will use the client id to fetch the local server.